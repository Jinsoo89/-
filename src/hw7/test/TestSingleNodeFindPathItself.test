# Find a path with Dijkstra's algorithm when there is a possible path

# Create graph players, which has a single node
CreateGraph players
AddNode players C.Ronaldo
AddEdge players C.Ronaldo C.Ronaldo 0.0

# Find path between the single node and itself
FindPath players C.Ronaldo C.Ronaldo