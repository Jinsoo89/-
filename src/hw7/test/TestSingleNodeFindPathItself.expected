# Find a path with Dijkstra's algorithm when there is a possible path

# Create graph players, which has a single node
created graph players
added node C.Ronaldo to players
added edge 0.000 from C.Ronaldo to C.Ronaldo in players 

# Find path between the single node and itself
path from C.Ronaldo to C.Ronaldo:
total cost: 0.000